perl.h

```diff
@@ -4505,6 +4505,8 @@ typedef        struct crypt_data {     /* straight from /usr/include/crypt.h */
 
 #include "perly.h"
 
+/* opaque struct type used to communicate between xop_dump and opdump_printf */
+struct OpDumpContext;
```

Comment:
This is a public (API) identifier without a `Perl_` prefix. Do we want that?

Reply:
We already have hundreds of those - to pick a random example, `typedef struct av AV` in perl.h. But you're right; no need to make it worse. I'll stick a prefix on it.